ui/components/multichain/account-list-menu/account-list-menu.js

```diff
@@ -173,6 +173,8 @@ export const AccountListMenu = ({ onClose }) => {
                   ({ origin }) => origin === currentTabOrigin,
                 );
 
+                const selected = selectedAccount.address === account.address;
```

Comment:
This change is just to improve code readability, and does not have any impact on the functionality of the fix, right @darkwing?

Reply:
Correct!

Reply:
Removed this change, not necessary for this PR.